src/components/Upload/Forms/DynamicForm.js

```diff
@@ -96,7 +96,7 @@ class DynamicForm extends Component {
 		this.setState({submitClicked: true});
 		let { validateFields } = this.props.form;
 		validateFields((err, values) => {
-			let newValues = values;
+			let newValues = JSON.parse(JSON.stringify(values).replace(/"\s+|\s+"/g,'"'));
```

Comment:
Add a check to isFormValid to verify search field isn't just a blank string after the trim